custom_components/unifi_access/hub.py

```diff
@@ -260,14 +272,64 @@ def on_message(self, ws: websocket.WebSocketApp, message):
                     )
                     if existing_door is not None:
                         existing_door.doorbell_request_id = None
+                        event = "doorbell_press"
+                        event_attributes = {
+                            "door_name": existing_door.name,
+                            "door_id": existing_door.id,
+                            "type": DOORBELL_STOP_EVENT,
+                        }
                         _LOGGER.info(
                             "Doorbell press stopped on %s Request ID %s",
                             existing_door.name,
                             doorbell_request_id,
                         )
-
+                case "access.logs.add":
+                    door = next(
+                        (
+                            target
+                            for target in update["data"]["_source"]["target"]
```

Comment:
Are we sure all the keys are in place here?

Reply:
Yes. It's part of the response schema and the keys are always there. I couldn't find a clean way to conditionally access properties in Python (short of using `get` followed by None check).